### PR TITLE
chore: update widget integration

### PR DESCRIPTION
**Pull request by shoom3301**
Replaced `@cowprotocol/widget-lib` by the latest version of `@cowprotocol/widget-react`

image

### CODE REVIEW

**Review by elena-zh — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]
One thing that I noticed.

Not cow.fi related of course.

I couldn't capture the screenshot in time, but the steps are as follows:
1. Pick as sell token one for you have no approvals and it's NOT permittable
2. Trigger the approval modal
3. Send the approval from the wallet
4. After approved, the approval toast is displayed, and it does not look nice

**Pull request merged**
